modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -63,23 +68,41 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
         } yield ObserveResult(id)
       }
 
-      Step[Action](
-        i,
-        None,
-        config.toStepConfig,
-        resources,
-        false,
-        (if(i == 0) List(List(toAction(systems.odb.sequenceStart(obsId, "").map(_ => Result.Ignored))))
-        else List())
-        ++
-        List(
-          sys.map(x => toAction(x.configure(config).map(y => Result.Configured(y.sys.name)))),
-          List(toAction(observe(config).map(x => Result.Observed(x.dataId))))
-        )
-        ++
-        (if(last) List(List(toAction(systems.odb.sequenceEnd(obsId).map(_ => Result.Ignored))))
-        else List())
-      )
+      extractStatus(config) match {
+        case "ready" =>
```

Comment:
Yes, perhaps there is even some constant on spModel. My rationale is that `ready` doesn't tell you much but some other constant could be clearer?